android/modules/network/src/java/ti/modules/titanium/network/NetworkModule.java

```diff
@@ -40,7 +40,7 @@
 public class NetworkModule extends KrollModule {
 
 	private static final String TAG = "TiNetwork";
-	private static CookieStore httpCookieStore;
+	private static java.net.CookieManager cookieManager;
```

Comment:
There's another class with the same name that is imported:-
```java import android.webkit.CookieManager; ```
I don't think I can import it.